### ISSUE TITLE
Error: All2HTML can't open default.tpl

### ISSUE THREAD

**Issue by Reikimann** [vimwiki/vimwiki#1193]
I'm using Neovim and Packer as my package manager.

When trying to convert my files to HTML
I get this message:

```
Vimwiki: Saving Vimwiki files ...

Vimwiki: Deleting non-wiki html files ...

Vimwiki: Converting wiki to html files ...
Vimwiki: Processing /home/my_user/vimwiki/index.wiki

Error detected while processing function vimwiki#html#WikiAll2HTML [45] ..<SNR>73_convert_file[110] ..<SNR>73_get_html_template: 

line 20:

E484: Can't open file /home/my_user/.local/share/nvim/site/pack/*/start/*/autoload/vimwiki/default.tpl

Press ENTER or type command to continue
```

The file is there, I have checked.
Pls help

**Comment by leyhline** [comment]
I just had the same problem. Seems to be a problem with expanding the * in the path. I use no plugin manager and just ran `git clone` using the default package mechanism.

I also quickly wrote a fix:

```patch
diff --git a/autoload/vimwiki/html.vim b/autoload/vimwiki/html.vim
index 0da3d81..ec0bda9 100644
--- a/autoload/vimwiki/html.vim
+++ b/autoload/vimwiki/html.vim
@@ -54,8 +54,9 @@ endfunction
 function! s:find_autoload_file(name) abort
   for path in split(&runtimepath, ',')
     let fname = path.'/autoload/vimwiki/'.a:name
-    if glob(fname) !=? ''
-      return fname
+    let match = glob(fname)
+    if match !=? ''
+      return match
     endif
   endfor
   return ''
```

I just not sure if this is 100% bulletproof since I don't have a good grasp of Vimscript. Maybe I can prepare a pull request nevertheless.

**Comment by tinmarino** [comment]
Fixed by #1195
